src/dvr/dvr_rec.c

```diff
@@ -82,6 +82,7 @@ dvr_rec_subscribe(dvr_entry_t *de)
   uint32_t rec_count, net_count;
   int ret = 0, pri, c1, c2;
   struct stat st;
+  int statRet;
```

Comment:
camelCase is not usually used for c programming, and doesn't really fit here either, use stat_ret instead.

Reply:
dONE